### DISCUSSION TITLE
Reduce the effect of gatekeeping within our dev community

### DISCUSSION THREAD

**Discussion by itrich** [SovereignCloudStack/issues#85] [Ideas]
Community,

we had some very good discussions in the first meetup of the newly formed Community SIG. We all agreed on the mission, that we urgently need to reduce the effect of gatekeeping and have to find a way for an easy/smooth on-boarding process into our dev community.

We identified several issues that need further investigation and brainstorming. Let's try a new format and discuss possibilities and solutions with [GitHub Discussion](https://github.com/SovereignCloudStack/issues/discussions).

### File sharing
While the Nextcloud offered a very good possibility to work together during the application phase, we now should come to a point where all our content is publicly available. Thus, meeting minutes, presentations or other material should be reachable where most of the development happens, i.e. GitHub.

### Polls and forms
We're using the Nextcloud for various services like polls and forms. Since the project team will still use a collaboration platform (either we're sticking to the existing Nextcloud or completely migrate to the Owncloud of the OSBA), these tools will remain available for the team. However, self-organized community members will have to use freely available services (e.g. [nuudel](https://nuudel.digitalcourage.de/)).

## Mailing lists
We're running Mailman 3, which syncs the list's members via [garloff/ucs2mailman3](https://github.com/garloff/ucs2mailman3) with our SCS UCS instance – the central identity management of our community platform. We should discuss how to maintain our mailing lists in the future. Any ideas?

## GitHub
The organization @SovereignCloudStack on GitHub consists of several teams. We should nominate one or two maintainers per team so that the participating companies can on- or off-board new colleagues on their own.

**Comment by fkr** [comment]
For the mailing lists: we can still use mailman3 but should actually use the real addresses of people instead of `firstname.lastname@scs.sovereignit.de` (and so far I failed to comprehend, why we do that).

Most of the  project related content that is currently in nextcloud is easily managed via github repos (and allows better collaboration without the burden of 'onboarding' (in the sense of account creation).

**Reply by berendt** [reply]
With the accounts on scs.sovereignit.de I think it is so because you then create an account via UCS, which automagically adds it in the Nextcloud and in Mailman and then the lists are subscribed directly. But would also prefer a direct subscribe.
